docs/src/literate-tutorials/linear_shell.jl

```diff
@@ -281,9 +283,9 @@ function integrate_shell!(ke, cv, qr_ooplane, X, data)
     ef1, ef2, ef3 = fiber_coordsys(p)
 
     for iqp in 1:getnquadpoints(cv)
-        N = cv.N[:,iqp]
-        dNdξ = cv.dNdξ[:,iqp]
-        dNdx = cv.dNdx[:,iqp]
+        N = [shape_value(cv, iqp, i) for i in 1:nnodes]
+        dNdξ = [shape_reference_gradient(cv, iqp, i) for i in 1:nnodes]
+        dNdx = [shape_gradient(cv, iqp, i) for i in 1:nnodes]
```

Comment:
Are you in the shell game? If so, do you know if we can improve the example with the proposed split of CellValues?

Reply:
I'm not really in that game... 
But it looks to me like using more of Ferrite's functionality should be possible for that particular example, but I don't see anything directly related to this split now.